react/__mocks__/vtex.checkout-resources/MutationUpdateSelectedAddress.ts

```diff
@@ -0,0 +1,13 @@
+import gql from 'graphql-tag'
+
+export default gql`
+  mutation MockUpdateSelectedAddress($address: AddressInput!) {
+    updateSelectedAddresS(input: $address) {
```

Comment:
Is the last s in uppercase (updateSelectedAddresS) a typo?

Reply:
yes! nice catch